cli/chrome-headless-render-pdf.js

```diff
@@ -58,6 +58,13 @@ if (typeof argv['chrome-binary'] === 'string') {
     chromeBinary = argv['chrome-binary'];
 }
 
+let chromeOptions = [];
+for (let i = 0; i < process.argv.length; i++) {
```

Comment:
Please use minimist for handling arguments

Reply:
IMHO `--chrome-option --no-sandbox` looks like both options are for parent command. And may be misleading. I guess just better indication. How to correctly pass additional arguments in help text is better solution.

Reply:
Fair point. I'd been quoting my params which made it more readable, but I think you're right.